node/malus/src/variants/suggest_garbage_candidate.rs

```diff
@@ -241,8 +280,30 @@ where
 	}
 }
 
+#[derive(Clone, Debug, clap::Parser)]
+#[clap(rename_all = "kebab-case")]
+#[allow(missing_docs)]
+pub struct SuggestGarbageCandidateOptions {
+	/// Determines the percentage of candidates that should be disputed. Allows for fine-tuning
+	/// the intensity of the behavior of the malicious node. Value must be betweeen 0-100.
+	#[clap(short, long, ignore_case = true, default_value_t = 100)]
```

Comment:
We should have some bounds for the value here and fail if value is out of bounds.

Reply:
Sorted with `clap::value_parser`:  https://github.com/paritytech/polkadot/pull/6066/commits/460ba61eb8e9869e9889f8fe52c6e7359d7b9d74